examples/quickstart/CMakeLists.txt

```diff
@@ -3,7 +3,13 @@
 #
 
 # TODO haven't quite decided the right way to run quickstart on Windows. Needs README update.
-if (NOT MSVC)
+#
+# Note: on macOS and other platforms, the 'command' described below may not work even if the cmake builds.
+# For example, it may be necessary to specify the sysroot, which CMake does, but the 'command' does not
+# handle such niceties. On a case-by-case basis it is fixable but it requires work that CMake knows how
+# to do but that is not trivial.
+#
+IF(${CMAKE_SYSTEM_NAME} MATCHES "Linux")
```

Comment:
I'm just confused here: I see a beginning IF() but no ENDIF().

Reply:
Maybe you missed that it remplaces the if(not visual studio)...

This part I might rework if I can find a way to make these tests work under macOS. Right now, they do not.

Reply:
But it has complicated interactions with, say, CMAKE_CXX_FLAGS.

Reply:
I'd want to make sure that it works with libc++ as well.

Reply:
Oh! OK! :+1: